webpack.config.js

```diff
@@ -70,7 +70,7 @@ if (EXTRACT || PRODUCTION) {
 }
 
 if (PRODUCTION) {
-    PLUGINS.push(...PROD_PLUGINS);
+    PLUGINS.push(PROD_PLUGINS);
```

Comment:
Not 100% sure if Webpack will find plugins defined in nested arrays (with this, `PLUGINS` will now be `[plugin, [plugin, plugin, ...], ...]` rather than `[plugin, plugin, plugin, ...]`), but I've opted to use the spread to flatten out the array to be safe.